packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -32,6 +32,44 @@ import 'ticker_provider.dart';
 /// (e.g., Android) that commonly use this type of overscroll indication.
 ///
 /// In a [MaterialApp], the edge glow color is the [ThemeData.accentColor].
+///
+/// When building a [CustomScrollView] with a [GlowingOverscrollIndicator], the
+/// indicator will apply to the entire scrollable area, regardless of what
+/// slivers the CustomScrollView contains, like a [SliverAppBar]. When building
```

Comment:
the "like a [SliverAppBar]" sentence fragment seems kinda lost. Maybe rephrase this a little?